src/components/Provider/index.tsx

```diff
@@ -15,23 +15,32 @@ const Base = styled(Grommet)`
 	color: ${(props) => px(props.theme.colors.text.main)};
 `;
 
-const Provider = ({ theme, ...props }: ThemedProvider) => {
+const defaultFontLink =
+	'https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,600;1,400&family=Ubuntu+Mono:wght@400;700&display=fallback';
+
+const Provider = ({
+	theme,
+	fontLinks = [defaultFontLink],
```

Comment:
I was thinking more about adding an `offlineFirst` property that we can use to make other decisions as well, such as not trying to load google maps, or captcha, for example. If it is offlineFirst, then we'll first try to load the locally available fonts if the user hasn't overridden the `font` theme property. If you are passing fontLinks, you might as well load them in your application.